src/annotator/guest.js

```diff
@@ -183,6 +183,12 @@ export default class Guest {
       source: 'guest',
     });
 
+    /**
+     * Integration that handles document-type specific functionality in the
+     * guest.
+     */
+    this._integration = createIntegration(this);
```

Comment:
Creation of the integration was moved to happen earlier because `this._connectSidebarEvents` now relies on it via calling `this.getDocumentInfo()`.

Reply:
Yes, I think so.